sass/layout/_sidebar-left.scss

```diff
@@ -458,6 +524,14 @@
         transform: rotate(-90deg);
         transition: transform 0.15s ease-out; /* should match collapse animation speed */
     }
+
+    .icon-flip {
+        -ms-filter: "progid:DXImageTransform.Microsoft.BasicImage(rotation=-1)";
```

Comment:
`-ms-filter`  shouldn't be needed anymore as MS Edge supports standardized transforms